### DISCUSSION TITLE
Problem setting up Whitebeet Carrier Board EVSE on Windows 7

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#108] [General]
Hi,
I'm having a problem getting the Whitebeet Carrier Board (EVSE) to work under Windows 7. Have installed 'scapy' and 'winpcap', but keep getting errors when I try to run the command setup command, i.e. 'python Application.py....'. The adapter name seems to be correct and the MAC Address for the host is also correct as far as I can tell.

I would be most grateful for any help you can provide with regard to this issue.

I've included a screenshot of the error. Best regards. Rob.

![beet-comms_1](https://user-images.githubusercontent.com/114229905/202393506-f0ccd54a-ab86-4543-925a-ac2fb04a907d.jpg)

**Comment by jpo-stx** [comment]
When I try to use non existant interface on my system (Windows 10 with npcap) I get a different error message.

Can you use try to use the name of the ethernet adapter which is shown in ipconfig:

```
PS C:\workspace\FreeV2G> ipconfig

Windows-IP-Konfiguration


Ethernet-Adapter Ethernet_onboard:

   Verbindungsspezifisches DNS-Suffix:
   Verbindungslokale IPv6-Adresse  . : ******
   IPv4-Adresse  . . . . . . . . . . : ******
   Subnetzmaske  . . . . . . . . . . : ******
   Standardgateway . . . . . . . . . : ******
```

In this case **Ethernet_onboard**
